### ISSUE TITLE
spin_once function to communicate with ROS

### ISSUE THREAD

**Issue by Hagar-Usama** [Das-Auto/SFND_Lidar_Obstacle_Detection#3]
> > you can do instead a function similar to the TTC requested one (spin_once) that runs one frame
> > which returns float[number_of_boxes_per_frame][6]
> 
> That is another task (for ROS integration) as mentioned at the last meeting? Michael is no longer blocked by me, right? @youssefMHanna 
> 

Yes Michael has already started this request is part of the ros integration task

_Originally posted by @youssefMHanna in https://github.com/Das-Auto/SFND_Lidar_Obstacle_Detection/issues/1#issuecomment-855172201_
